### PR TITLE
Feature: Looker permission set

### PR DESCRIPTION
**Pull request by Vitzeno**
# Description

This PR allows a user to create a `looker_permission_set` by supplying a name and a set of permisions. This does not bind the new created `looker_permission_set` to any role as of yet.

### CODE REVIEW

File: docs/resources/permission_set.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+page_title: "looker_permission_set Resource - terraform-provider-looker"
+subcategory: ""
+description: |-
+  Manages a Looker permission set
+---
+
+# looker_permission_set (Resource)
+
+Manages a Looker permission set
+
```
Comment (jessicasomaiya) [inline docs/resources/permission_set.md:12]: To keep consistent with other docs, you could add a 

`Example Usage` section here with the example you have in examples/resources/permission_set/main.tf :)

Comment (Vitzeno) [reply docs/resources/permission_set.md:12]: good point, not sure why the generator missed it unless it's supposed to be manual?

Comment (jessicasomaiya) [reply docs/resources/permission_set.md:12]: I've been doing it manually :'( there's a ticket to configure the generator to massively improve the dev experience!!

File: internal/conv/conv.go [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
 func PString(s string) *string {
 	if s == "" {
 		return nil
 	}
 	return &s
 }
 
+func PSlices(s *schema.Set) *[]string {
```
Comment (jessicasomaiya) [inline internal/conv/conv.go:12]: there's a function already called `schemaSetToSliceString` which you could use which converts a set to a `[]string`. You could use this func or move it to somewhere general to use it? 

https://github.com/resolutionlife/terraform-provider-looker/blob/main/looker/resource_looker_user_role.go#L209

Comment (Vitzeno) [reply internal/conv/conv.go]: forgot about that one, I've moved it to the conv package along side a generic PSlices function

Comment (Vitzeno) [reply internal/conv/conv.go]: though I'm still unsure if that's the best place for it, since everything else in that package is about returning pointer?

Comment (jessicasomaiya) [reply internal/conv/conv.go]: Yep I'd agree - I'll ask around to see where is best to put this. Maybe in a helpers?

Comment (jessicasomaiya) [reply internal/conv/conv.go]: maybe a util package or something hidden inside `internal`?

**Review by jessicasomaiya — APPROVED** [review]
Looks great Mo! 👍🏾

**Pull request merged**